### PR TITLE
lowercases all pubsubName in HTTP API invocations

### PR DESCRIPTION
**Pull request by mchmarny**
Part of a larger effort to address `pubsubname` REST parameter inconsistencies across the Dapr org repos.  

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: docs/release_notes/v0.10.0.md [hunk 1]
```diff
@@ -421,14 +421,14 @@ app.get('/dapr/subscribe', (req, res) => {
         {
             topic: "newOrder",
             route: "orders",
-            pubsubName: "messagebus"
+            pubsubname: "messagebus"
```
Comment (artursouza) [inline docs/release_notes/v0.10.0.md:424]: This is case sensitive, have you tried it? Also, we are using camelCase, not?

**Review by artursouza — CHANGES_REQUESTED** [review]

File: docs/release_notes/v0.10.0.md (hunk 1 above)
Comment (mchmarny) [reply docs/release_notes/v0.10.0.md:424]: this works either way, this is for consistency across all the repos in the Dapr org.

Comment (mchmarny) [reply docs/release_notes/v0.10.0.md:424]: Tehcnicallty, this was in release notes for the version we are not patching so maybe leave this file as is

Comment (artursouza) [reply docs/release_notes/v0.10.0.md:424]: Are we moving away from camelCase? `newOrder` here is still in camelCase.

Comment (mchmarny) [reply docs/release_notes/v0.10.0.md:424]: The [API](https://github.com/dapr/docs/blob/master/reference/api/pubsub_api.md#http-response-body) has to be the source of truth there, no?

Comment (artursouza) [reply docs/release_notes/v0.10.0.md:424]: The documentation is mixed. The code should be the source of truth and it uses camelCase. I agree to bring consistency - which seems to be to stick to camelCase.

https://github.com/dapr/dapr/blob/ce89980eda9cc86efbc11a745e2959b1925236ca/pkg/runtime/runtime.go#L73

Comment (mchmarny) [reply docs/release_notes/v0.10.0.md:424]: paging @yaron2, I know he had some opinions on this topic

Comment (mchmarny) [reply docs/release_notes/v0.10.0.md:424]: > The documentation is mixed

Only a very few places had this camelCase BTW. Also, worth pointing out that this is for REST API only. The protos have number of places where they differ from the HTTP API docs.

Comment (yaron2) [reply docs/release_notes/v0.10.0.md:424]: When I tested this, both `pubsubname` and `pubsubName` were valid.

Comment (yaron2) [reply docs/release_notes/v0.10.0.md:424]: @artursouza this is the correct place: https://github.com/dapr/dapr/blob/4e3198ca8d84184b77083674a2018ac2ad76b484/pkg/runtime/pubsub/subscription.go#L4

Comment (yaron2) [reply docs/release_notes/v0.10.0.md:424]: The reference you gave is about internal metadata that dapr sets for itself, and unrelated to the `pubsubname` the user sets.

**Comment by youngbupark** [comment]
Please do not change release note which we already released. new change needs to go in the next release note...

**Comment by mchmarny** [comment]
@youngbupark 

> Technicality, this was in release notes for the version we are not patching so maybe leave this file as is

File: docs/release_notes/v0.10.0.md (hunk 1 above)
Comment (artursouza) [reply docs/release_notes/v0.10.0.md]: In this example, we also need to fix "newOrder" to "neworder"

**Review by artursouza — APPROVED** [review]

**Pull request merged**